### DISCUSSION TITLE
Popup + Formulário

### DISCUSSION THREAD

**Discussion by ghost** [frontendbr/forum#239] [Perguntas e Respostas]
Olá galera bom dia,

Quero fazer um `<a href="exemplo.com"> Link </a>` que abra um `Popup` sem barra de endereços e com um formulário para preencher.

**Comment by felipefialho** [comment]
@lucasbicalleto16 Procure "lightbox" ou "modal" no Google ;). É o que você quer.

**Comment by felipemrodrigues** [comment]
@lucasbicalleto16 Fiquei na dúvida se você quer um popup default do browser ou outra solução como @LFeh citou.

Pode deixar mais claro?

**Comment by ghost** [comment]
@felipemrodrigues  Seria como @LFeh  Citou, mas não obtive sucesso ainda.!

**Comment by felipefialho** [comment]
@lucasbicalleto16 Tem 1 milhão de plugins prontos, e é super fácil criar o seu caso não queira usar de terceiros.

[Pode usar até CSS se quiser](http://www.felipefialho.com/css-components/#component-modal)

**Comment by notnotgabriel** [comment]
Eu gosto do http://vodkabears.github.io/remodal/ :)

**Comment by bernardodiasc** [comment]
Esse modal do @LFeh é um belo exemplo. Aliás, parabéns por todos Pure CSS Components, muito maneiro.

Mas acho que o que @ghost está precisando é mais pro lado do http://www.w3schools.com/jsref/met_win_open.asp
